### DISCUSSION TITLE
Can't watch HDR content even with the recomendad USB Grabber

### DISCUSSION THREAD

**Discussion by javierpal** [awawa-dev/HyperHDR#795] [Q&A]
Hey I wanted to use HyperHDR, because most of the PS5 and Series X games are HDR, and some movies too, I want to be able to use my ambilight system with those, but for some reason, none of my USB Grabbers can pick up HDR content, only DOBLY VISION.

here is my Setup:

HDMI Switch : Orei 8k 4x1 HDMI 2.1 Swtich https://www.amazon.com.mx/gp/product/B0BSB6C4RR/ref=ppx_yo_dt_b_asin_title_o08_s00?ie=UTF8&th=1
HDMI Divisor: Ezcoo HDMI 1x2 4K 120Hz 4:4:4 HDR 8K 60Hz VRR G/Sync CEC EDID Switch https://www.amazon.com.mx/dp/B0B7W94LHB?psc=1&ref=ppx_yo2ov_dt_b_product_details
Capture Cards are USB Graber with the chip MS2109 (2 of them because I wanted to test) and  another one, MS2130, none of those grab the HDR signal
And of course A raspberry pi 4 with HyperHDR installed, using USB3.0 port.

the ezcoo divisor is set to downscale signal to 1080p@60hz or 120hz depending on input.
on TV Dobly Atmos Dobly Vision 4k@120hz HDR10+ working fine.
HyperHDR working with almost anything but HDR content, so no games/prime video movies

Do you know what the problem could be? do I need to downgrade my usb graber (MS2130) firmware?

Maybe I have to switch to ElGatoHD60 X so I can grab the HDR signal?

**Comment by awawa-dev** [comment]
Hi
My equipment doesnt support HDR10+ so I cant test it and help here. Beside ms2130 can capture only some selected format at 1080p120 due to limited bandwidth. And no VRR for sure (HDMI splitter wont change VRR to constant framerate for one output).

**Reply by javierpal** [reply]
Is there a way to help? I'm a developer, but I don't have signal generator or stuff like that but maybe I can do something (I like electronics).

**Reply by javierpal** [reply]
Maybe I need to find a way to mirror the image but without all the HDR+ and VRR stuff.

**Reply by awawa-dev** [reply]
Usually Ezcoo can enforce LLDV mode on the source and that protocol can be captured by ms2130. But I dont know this splitter model. And no downscale is possible at all even with Ezcoo, if the source and TV selected different closed proprietary protocol and HDMI splitter allowed it (via EDID). For VRR you would need for example Elgato HD60x but it rather wont change dramatically situation here (beside VRR it may help with 1080p120 compatibility). I have some expectations regarding HDR10+, but they are only based on theory. In practice it may be different. The biggest change from our point of view is the dynamic brightness of the scene.

Try to test the setup using 4k60 HDR10 because the problem may be hardware (usually HDMI splitter fault). And HDR10 must work.

**Reply by awawa-dev** [reply]
btw according to the spec max what it can offer is just `4K@120Hz 4:4:4 8bit HDR` not HDR10+, the latter requires at least 10 bits.

**Reply by javierpal** [reply]
Weird my TV is showing HDR10 with 10 bits when  playing games and watching prime video movies
